tailcfg/tailcfg.go

```diff
@@ -1341,6 +1341,9 @@ const (
 	PeerCapabilityWakeOnLAN PeerCapability = "https://tailscale.com/cap/wake-on-lan"
 	// PeerCapabilityIngress grants the ability for a peer to send ingress traffic.
 	PeerCapabilityIngress PeerCapability = "https://tailscale.com/cap/ingress"
+	// PeerCapabilityWebUI grants the ability for a peer to edit features from the
+	// device web ui.
```

Comment:
Web UI

Reply:
Actually, I think the value here also should be _just_ `tailscale.com/cap/webui`, not `https://tailscale.com/cap/webui`. Looking at the other similar caps, some of them seem to be defined in their own packages rather than `tailcfg`. Is this a legacy list?

Reply:
Updated to remove the "https://" prefix, but kept in tailcfg